src/deprecated.jl

```diff
@@ -357,3 +357,85 @@ function by(d::AbstractDataFrame, cols::Any; sort::Bool=false, skipmissing::Bool
 end
 
 @deprecate groupvars(gd::GroupedDataFrame) groupcols(gd)
+
+export aggregate
+
+function aggregate(d::AbstractDataFrame, f::Any; sort::Bool=false)
+    df = select(d, names(d) .=> [f])
+    if sort
+        Base.depwarn("`aggregate` is deprecated. " *
+                     "Instead use `sort!(select(d, names(d) .=> f))`.", :aggregate)
+        sort!(df)
+    else
+        Base.depwarn("`aggregate` is deprecated. " *
+                     "Instead use `select(d, names(d) .=> f)`.", :aggregate)
+    end
+    return df
+end
+
+function aggregate(d::AbstractDataFrame, fs::AbstractVector; sort::Bool=false)
+    df = hcat([select(d, names(d) .=> [f]) for f in fs]..., makeunique=true)
+    if sort
+        Base.depwarn("`aggregate` is deprecated. Instead" *
+                     " use `sort!(select(d, [names(d) .=> f for f in fs]...))` " *
+                     "if functions in `fs` have unique names.", :aggregate)
+        sort!(df)
+    else
+        Base.depwarn("`aggregate` is deprecated. Instead" *
+                     " use `select(d, [names(d) .=> f for f in fs]...)` if functions " *
+                     "in `fs` have unique names.", :aggregate)
+    end
+    return df
+end
+
+function aggregate(gd::GroupedDataFrame, f::Any; sort::Bool=false)
+    df = combine(gd, valuecols(gd) .=> [f])
+    if sort
+        Base.depwarn("`aggregate` is deprecated. Instead use" *
+                     "sort!(combine(gd, valuecols(gd) .=> f), setdiff(names(df), groupcols(gd)))`.",
```

Comment:
`df` doesn't exist for the user. Maybe break this into two steps separated by `;`.

Also:
```suggestion
                     " `sort!(combine(gd, valuecols(gd) .=> f), setdiff(names(df), groupcols(gd)))`.",
```